fe/fe-core/src/test/java/com/starrocks/pseudocluster/PseudoCluster.java

```diff
@@ -220,7 +220,7 @@ public void runSqlList(String db, List<String> sqls) throws SQLException {
     }
 
     public void runSqls(String db, String... sqls) throws SQLException {
-        runSqlList(db, Arrays.stream(sqls).collect(Collectors.toList()));
+        runSqlList(db, Arrays.stream(sqls).collect(Collectors.toList()), true);
```

Comment:
What's the difference between the runSqlList and runSqls?

Reply:
verbose=true will print executed sql and execution time in system.out